deployment/README.md

```diff
@@ -65,14 +73,20 @@ the browsers and so cannot be attached to another instance in read-write mode.
 5. Install Hail.
 
    ```
-   apt-get -qq install wget software-properties-common
-   wget -qO - https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public | apt-key add -
-   add-apt-repository --yes https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/
-   apt-get -qq update
-   apt-get -qq install adoptopenjdk-8-hotspot
+   apt-get -qq update && \
```

Comment:
since this install isn't happening within the context of docker (from what I can tell?), you don't *need* to chain everything with an `&&`, or clean up the /var/lib/apt/lists folder -- those are dockerisms that are useful for keeping the final image small

Reply:
Aha, ok that makes sense. Since the GCE vm will be deleted anyway. Thanks as always for the context around little bits and pieces that I'm more or less mime-ing from other places.